Source/WebCore/layout/integration/inline/LayoutIntegrationLineLayout.cpp

```diff
@@ -1003,11 +1003,18 @@ void LineLayout::shiftLinesBy(LayoutUnit blockShift)
     for (auto& line : m_inlineContent->displayContent().lines)
         line.moveInBlockDirection(blockShift, isHorizontalWritingMode);
 
-    for (auto& box : m_inlineContent->displayContent().boxes) {
+    LayoutUnit dx = (isHorizontalWritingMode) ? 0_lu : blockShift;
+    LayoutUnit dy = (isHorizontalWritingMode) ? blockShift : 0_lu;
```

Comment:
please use deltaX, deltaY according to WebKit coding style: https://webkit.org/code-style-guidelines/#names-full-words.

Reply:
Also can we remove the redundant brackets around `isHorizontalWritingMode` ?

Reply:
Done.